doc/hardware/porting/board_porting.rst

```diff
@@ -361,21 +519,28 @@ named ``plank``:
 
      endif # BOARD_PLANK
 
-:file:`plank_defconfig`
+:file:`plank_defconfig` / :file:`plank_<identifier>_defconfig`
   A Kconfig fragment that is merged as-is into the final build directory
   :file:`.config` whenever an application is compiled for your board.
 
-  You should at least select your board's SOC and do any mandatory settings for
-  your system clock, console, etc. The results are architecture-specific, but
-  typically look something like this:
+  If both the common :file:`plank_defconfig` file and one or more board
+  identifier specific :file:`plank_<identifier>_defconfig` files exist, then
```

Comment:
might want to mention revision also

Reply:
No, let's keep revisions out of here, just as it is on main. 

If this guide is also intended for someone making their first board then let's stick to this.
It's complex enough already.

The revision section describes the additional files for anyone in need of extending a board with more revisions.